.github/workflows/backport.yml

```
@@ -0,0 +1,22 @@
+---
+name: Backport Assistant Runner
+
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    container: hashicorpdev/backport-assistant:0.2.2
+    steps:
+      - name: Run Backport Assistant
+        run: |
+          backport-assistant backport
+        env:
+          BACKPORT_LABEL_REGEXP: "(?P<target>\\d+\\.\\d+\\.x)-backport"
```

It [looks like](https://github.com/hashicorp/consul-api-gateway/labels?q=backport) we've used the inverse of this already - `backport-0.2.x` - but obviously we can create a new standard if we want. Just calling this out

```suggestion
          BACKPORT_LABEL_REGEXP: "backport-(?P<target>\\d+\\.\\d+\\.x)"
```

---

Using a shared prefix like `backport` groups the labels together in the selector which is nice. FWIW Consul core uses `backport/` https://github.com/hashicorp/consul/labels